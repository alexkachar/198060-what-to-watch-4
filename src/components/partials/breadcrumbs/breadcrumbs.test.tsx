import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';

import Breadcrumbs from './breadcrumbs';

const MOVIE_TITLE = `The Grand Budapest Hotel`;
const MOVIE_ID = `1`;

it(`Breadcrumbs renders correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <Breadcrumbs
            id={MOVIE_ID}
            movieTitle={MOVIE_TITLE} />
        </BrowserRouter>, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
