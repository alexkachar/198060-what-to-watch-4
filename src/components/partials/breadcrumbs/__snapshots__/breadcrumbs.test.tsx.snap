// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders correctly 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="/film/1"
        onClick={[Function]}
      >
        The Grand Budapest Hotel
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <p
        className="breadcrumbs__link"
      >
        Add review
      </p>
    </li>
  </ul>
</nav>
`;
